core/src/org/pentaho/di/core/database/PostgreSQLDatabaseMeta.java

```diff
@@ -150,6 +150,11 @@ public boolean needsToLockAllTables() {
     return false;
   }
 
+  @Override
+  public String getSQLListOfSequences() {
+    return "SELECT relname AS sequence_name FROM pg_statio_all_sequences";
```

Comment:
I have no problem changing this, but should I also update getSQLSequenceExists() to be consistent?